stable/nginx-ingress/values.yaml

```diff
@@ -216,12 +216,12 @@ controller:
   #    - name: copy-portal-skins
   #      mountPath: /srv/var/lib/lemonldap-ng/portal/skins
 
-  extraVolumeMounts: {}
+  extraVolumeMounts: []
   ## Additional volumeMounts to the controller main container.
   #  - name: copy-portal-skins
   #   mountPath: /var/lib/lemonldap-ng/portal/skins
```

Comment:
should the mountPath be aligned with the name above?

Reply:
@davidkarlsen It probably should, but it's in a comment and unrelated to the PR.

Reply:
Do you want me to fix that comment change in this PR?

Reply:
it's a bit nit-picking - but I think it is worthwhile - and easier to avoid error if people uncomment to use it.